imagemagick.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "magick",
+    "buildsystem": "simple",
```

Comment:
this should be autotools buildsystem.

Reply:
is this just a simple replacement? build-commands remain the same?

Reply:
no, there is no build commands in that case. use `config-opts` for the configure options.